source/frontend/src/features/mapSideBar/subdivision/AddSubdivisionContainer.tsx

```diff
@@ -56,7 +56,7 @@ const AddSubdivisionContainer: React.FC<IAddSubdivisionContainerProps> = ({ onCl
 
   const getAddress = useCallback(
     async (pid: string): Promise<AddressForm | undefined> => {
-      const bcaSummary = await getPrimaryAddressByPid(pid, 3000);
+      const bcaSummary = await getPrimaryAddressByPid(pid);
```

Comment:
did you mean to delete the timeout here? the other changes put the timeout at 30000ms but here it gets removed - why?

Reply:
I aligned it with the rest of the additions, so 30s timeout